Decred Wallet/Features/Send/SendCompletedViewController.swift

```diff
@@ -47,9 +47,9 @@ class SendCompletedViewController: UIViewController {
     
     @objc func copyTxHash() {
         UIPasteboard.general.string = self.transactionHash
-        self.hashHeaderLabel.text = "HASH: (copied)"
+        self.hashHeaderLabel.text = LocalizedStrings.copyHash
```

Comment:
`LocalizedStrings.hashCopied`
`copyHash` sounds like you're requesting an action, instead of stating that the action has already taken place